core/tests/protractor_utils/ExplorationEditorPage.js

```diff
@@ -101,7 +101,7 @@ var ExplorationEditorPage = function() {
   /*
    * Workflows
    */
```

Comment:
Should the linter cover these kinds of comments as well?

Reply:
NO